config/filesystems.php

```diff
@@ -57,6 +57,13 @@
             'url' => site_url(),
             'visibility' => 'public',
         ],
+        
+        'sage' => [
+            'driver' => 'local',
+            'root' => get_theme_file_path(),
+            'url' => site_url(),
```

Comment:
This should be `get_theme_file_uri()` to be consistent with `get_theme_file_path()`.